package Protos;

import "base.proto";

message WhiteNoiseInputC {
    required double mean = 1;
    required double sd = 2;
    required double T = 3;
}

message WhiteNoiseInputState {
    required double rng_state = 1;
    required bool cache = 2;
}

